src/modules/Game.class.js

```diff
@@ -20,25 +39,114 @@ class Game {
    * If passed, the board will be initialized with the provided
    * initial state.
    */
-  constructor(initialState) {
-    // eslint-disable-next-line no-console
-    console.log(initialState);
+  constructor(initialState = Game.deafultInitialState, isForCheck = false) {
```

Comment:
The constructor parameter 'initialState' is defaulting to 'Game.deafultInitialState', which contains the typo mentioned earlier. After fixing the typo, this should be updated to 'Game.defaultInitialState'.